hardware/arduino/avr/cores/arduino/USBCore.cpp

```diff
@@ -72,18 +65,14 @@ const u8 STRING_PRODUCT[] PROGMEM = USB_PRODUCT;
 const u8 STRING_MANUFACTURER[] PROGMEM = USB_MANUFACTURER;
 
 
-#ifdef CDC_ENABLED
```

Comment:
I'd also not force the CDC to be automatically added. This is not pluggable if you ask me. There should be a way to remove it and use other functions for usb. And you should add weak events for the CDC Serial as in my HID Project.